### PR TITLE
Fix initial value for KernelPickerComponent, move PROS menu to root menu

### PR DESCRIPTION
**Pull request by edjubuh**
### Summary

- Fix initial value for KernelPickerComponent
- Move PROS menu to root menu
- FolderPickerComponent now respects a path manually typed/pasted into the text editor.

### Test Plan

- [X] New project dialog immediate shows "latest" as selected option, not waiting for CLI to come back with the list of kernels
- [X] PROS menu is now at root level, more accessible for users. We're a full blown IDE usually installed with PROS Editor, so we can have our own menu.

#### References

N/A

### CODE REVIEW

File: lib/components/pros/FolderPickerComponent.js [hunk 1]
```diff
@@ -4,15 +4,14 @@
...
+export default class FolderPickerComponent {
   constructor(props) {
     this.props = props
 
     this.onPickDir = cmp => event => {
-      const { onChanged } = cmp.props
       atom.pickFolder(paths => {
         if (paths && paths[0]) {
-          onChanged(paths[0])
+          this.props.onChanged(paths[0])
```
Comment (HotelCalifornia) [inline lib/components/pros/FolderPickerComponent.js]: Why was this necessary?

Comment (HotelCalifornia) [reply lib/components/pros/FolderPickerComponent.js]: Also, if you're going to remove references to `cmp` I'd rather you just remove it as a formal parameter altogether

Comment (edjubuh) [reply lib/components/pros/FolderPickerComponent.js]: I don't think it's necessary, but it improves readability. It took me 10-15 minutes to figure out what was going on, but removing the parameter entirely has the same functionality. Having the parameter doesn't give you anything.

Comment (HotelCalifornia) [reply lib/components/pros/FolderPickerComponent.js]: suffice it to say, there was a reason I did it this way. that said, if you can confirm it works the other way, go for it.

Comment (edjubuh) [reply lib/components/pros/FolderPickerComponent.js]: It works the way proposed.

File: lib/components/pros/FolderPickerComponent.js [hunk 2]
```diff
@@ -4,36 +4,43 @@
...
 import { TextEditor } from 'atom'
 
-export default class FodlerPickerComponent {
+export default class FolderPickerComponent {
   constructor(props) {
     this.props = props
 
-    this.onPickDir = cmp => event => {
-      const { onChanged } = cmp.props
+    this.onPickDir = () => event => {
```
Comment (HotelCalifornia) [inline lib/components/pros/FolderPickerComponent.js]: if the arrow functions are confusing, let me put it this way:
the code there is now equivalent to
```js
function onPickDir() {
    return function(event) {
        // etc
     }
}
```

Comment (HotelCalifornia) [reply lib/components/pros/FolderPickerComponent.js]: the better way to refactor for simplicity would be

```js
this.onPickDir = event => { ... }
...
<button class='btn btn-default' onclick={this.onPickDir}>
```

**Review by HotelCalifornia — APPROVED** [review]
